meta-xt-dom0/recipes-core/images/core-image-thin-initramfs.bb

```diff
@@ -4,7 +4,7 @@ configured. RAM-based Initial Root Filesystem (initramfs) is used to avoid \
 need for accessing block/mtd devices. Shall contain kernel images for other \
 VMs."
 
-IMAGE_INSTALL = "packagegroup-core-boot ${CORE_IMAGE_EXTRA_INSTALL}"
+IMAGE_INSTALL = "packagegroup-core-boot ${CORE_IMAGE_EXTRA_INSTALL} xen-tools-xenstat"
```

Comment:
I'm not sure that this is a good place to add.
For example, we have https://github.com/xen-troops/meta-xt-common/blob/master/meta-xt-control-domain/recipes-core/images/core-image-thin-initramfs.bbappend#L1
And may be it's better to add xenstt to that append.

So, we need opinion from @lorc